src/javascript/app_2/App/Containers/Routes/routes.jsx

```diff
@@ -0,0 +1,28 @@
+import { PropTypes as MobxPropTypes } from 'mobx-react';
+import PropTypes                      from 'prop-types';
+import React                          from 'react';
+import { withRouter }                 from 'react-router';
+import ErrorComponent                 from '../../Components/Elements/Errors';
+import BinaryRoutes                   from '../../Components/Routes';
+import { connect }                    from '../../../Stores/connect';
+
+const Routes = (props) => {
+    if (props.has_error) {
```

Comment:
in the future, perhaps can add check for `is_loading` and load `<Loading />` here instead of adding it individually in each page